tests/common/experimental/test_reducers_and_aggregators.py

```diff
@@ -146,24 +217,28 @@ def test_shape_aggregator(self):
         assert aggregator._collected_samples == 1
         assert ref_shape == aggregator.aggregate()
 
-    def test_min_max_aggregators(self, tensor_processor):
-        min_aggregator = MinAggregator(tensor_processor)
-        max_aggregator = MaxAggregator(tensor_processor)
+    @pytest.mark.parametrize(
+        "aggregation_axes,min_ref,max_ref",
+        OFFLINE_AGGREGATORS_TEST_CASES,
+    )
+    def test_min_max_aggregators(self, aggregation_axes, min_ref, max_ref, tensor_processor):
+        min_aggregator = MinAggregator(tensor_processor=tensor_processor, aggregation_axes=aggregation_axes)
+        max_aggregator = MaxAggregator(tensor_processor=tensor_processor, aggregation_axes=aggregation_axes)
         input_ = np.arange(3 * 3).reshape((1, 3, 3))
         input_[0, 0, 0] = -10000
         for i in range(-5, 5):
             min_aggregator.register_reduced_input(self.get_nncf_tensor(input_ * (-i)))
             max_aggregator.register_reduced_input(self.get_nncf_tensor(input_ * i))
 
-        min_ref = [[[-50000, -4, -8], [-12, -16, -20], [-24, -28, -32]]]
-        assert self.all_close(min_ref, min_aggregator.aggregate())
-
-        max_ref = [[[50000, 4, 8], [12, 16, 20], [24, 28, 32]]]
-        assert self.all_close(max_ref, max_aggregator.aggregate())
+        assert self.all_close(
+            min_aggregator.aggregate(),
+            min_ref,
+        )
+        assert self.all_close(max_aggregator.aggregate(), max_ref)
 
     NO_OUTLIERS_TEST_PARAMS = [
         (MeanAggregator, True, 1, 1404.5138888888905),
-        (MedianAggregator, True, 1, 15.5),
+        (MedianAggregator, True, 1, 24.0),
```

Comment:
Is it really necessary to change the test case? Or there is some kind of bug hidden?

Reply:
There may be a bug and a misalignment in median calculation specifically since, as I discovered in course of #2120, `torch.median` has a different [behaviour](https://pytorch.org/docs/stable/generated/torch.median.html) for sets with even element count from the [numpy behaviour](https://numpy.org/doc/stable/reference/generated/numpy.median.html). 

In #2120 I brought the torch code in alignment with the rest of the code using numpy, need to check this case carefully to see if we are hitting this kind of bug here.

Reply:
I did it to have different results for MedianAggregator and MedianNoOutliersAggregator as previously they we the same due. I've changed amount of registered statistics here only for this case
https://github.com/openvinotoolkit/nncf/pull/2117/files/9618b499c91fb7d86055db1bcdc6c8f83ecf6424#diff-2b321b1866d8d8bb42cc6d1aed79c7a4957d9f64cf4e64133a09a3375eff7546R301-R304